{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/.tx/config

```diff
@@ -0,0 +1,14 @@
+[main]
```

Comment:
Maybe add some comments in this file explaining that you should add in your credentials here, when you need them, etc? Do you need your creds to pull, or only to push?